src/model_rng.hpp

```diff
@@ -162,13 +162,37 @@ class bs_model {
    * @param[in] jacobian `true` to include Jacobian adjustment for
    * constrained parameter transforms
    * @param[in] theta_unc unconstrained parameters
-   * @param[in,out] val log density produced
-   * @param[in,out] grad gradient produced
-   * @param[in,out] hess Hessian produced
+   * @param[out] val log density produced
+   * @param[out] grad gradient produced
+   * @param[out] hess Hessian produced
    */
   void log_density_hessian(bool propto, bool jacobian, const double* theta_unc,
                            double* val, double* grad, double* hessian) const;
 
+  /**
+   * Calculate the log density and the product of the Hessian with the specified
+   * vector for the specified unconstrain parameters and write it into the
+   * specified value pointer and Hessian-vector product pointer, dropping
+   * constants it `propto` is `true` and including the Jacobian adjustment if
+   * `jacobian` is `true`.
+   *
+   * @note If `BRIDGESTAN_AD_HESSIAN` is not defined, the complexity of this
+   * function goes from O(N^2) to O(N^3), and the accuracy of the result is
```

Comment:
Is this correct?  If `BRIDGESTAN_AD_HESSIAN` is not defined, then we're using an approximation.  The approximation should be faster with reduced accuracy, right?

Reply:
This was true when we were calculating the Hessian and then multiplying, I just missed this when I updated the other docs after the introduction of `finite_diff_hessian_times_vector_auto`